testing/sdk_utils.py

```diff
@@ -169,3 +170,47 @@ def sort(coll):
 def invert_dict(d: dict) -> dict:
     """ Returns a dictionary with its values being its keys and vice-versa. """
     return dict((v, k) for k, v in d.items())
+
+
+# https://github.com/pytoolz/toolz/blob/2bd9139d0d0e17d3426cb467b5f58b1fb6d8a439/toolz/itertoolz.py#L791
+def getter(index):
```

Comment:
Why not just depend on `pytoolz/toolz` directly?  Also, I don't know about having these functions here in `sdk_utils`. The module is already a little too general, but at least it's *mostly* limited to DC/OS specifics.

Reply:
I created a JIRA under the "technical tasks" epic for this: https://jira.mesosphere.com/browse/DCOS-42000